### PR TITLE
Make tests and make them pass on Travis and AppVeyor

### PR DESCRIPTION
**Pull request by ftomassetti**
This is a big, disorganized patch.
As first thing we removed a bunch of files and imported plate-tectonics as a module.
Then we wrote super basic tests just for sanity check.
We configured AppVeyor and Travis and we have those tests passing.

This should be the basis for starting having a decent process from now on :)

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "plate-tectonics"]
+	path = plate-tectonics
+	url = https://github.com/Mindwerks/plate-tectonics.git
```
Comment (psi29a) [inline .gitmodules]: We doing submodules now?  I've always had painful experiences here. Is this for appveyer?

File: README.md [hunk 2]
```diff
@@ -17,12 +17,23 @@ Requirements
 ============
 
 Supported versions:
-* Python 2.6
+* Python 2.6 (currently not supported on AppVeyor)
```
Comment (psi29a) [inline README.md]: 2.6 is only really supported by RHEL these days. I don't mind dropping support for it.

File: .gitmodules (hunk 1 above)
Comment (ftomassetti) [inline .gitmodules]: I have not experience with submodules. The problem is that it makes sense to have plate-tectonics has a C library, independent by any python code. We want then to wrap it in a python module but to do that we need its code. Can we do in another way?

File: README.md (hunk 2 above)
Comment (ftomassetti) [inline README.md]: It does not cost us much effort to support it, the only problem is finding out how to configure AppVeyor with the right compiler (we should compile the python wheels with the same compiler used to compile that version of the python interpreter)

**Comment by ftomassetti** [comment]
This pull request has been superseded by moving this code to plate-tectonics, closing.

**Pull request closed without merging**